woocommerce/woocommerce-ios
# Setup Semantic POS colors

#12702


## Description


- Basic setup for semanting POS color
- There will be other PR(s) regarding this for light/dark colors, other colors etc.

## Testing instructions


- open POS temporary view and check that the colors are applied
- just make sure CI passes

## Screenshots


![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-05-14 at 10 12 02](https://github.com/woocommerce/woocommerce-ios/assets/6242034/47c75efb-2821-4392-bf02-43c94b5488f2)


---
